samples/sampler/stories/qa/Panels.js

```diff
@@ -0,0 +1,225 @@
+
+Panels.displayName = 'Panels';
+const Config = mergeComponentMetadata('Panels', Panels);
+
+const BasicPanels = () => {
+	const [index, setIndex] = React.useState(0);
+	const goNext = () => setIndex(clamp(0, 2, index + 1));
+	const goPrevious = () => setIndex(clamp(0, 2, index - 1));
+	const goFirst = () => setIndex(0);
+	const goThird = () => setIndex(2);
+
+	return (
+		<Panels
+			index={index}
+			noAnimation={boolean('noAnimation', Config, false)}
+			noCloseButton={boolean('noCloseButton', Config, false)}
+			orientation={select('orientation', ['horizontal', 'vertical'], Config)}
+		>
+			<Panel>
+				<Header title="First Panel" />
+				<Item onClick={goNext}>Second</Item>
+				<Item onClick={goThird}>Third</Item>
+			</Panel>
+			<Panel>
+				<Header title="Second Panel" />
+				<Item onClick={goPrevious}>First</Item>
+				<Item onClick={goNext}>Third</Item>
+			</Panel>
+			<Panel>
+				<Header title="Third Panel" />
+				<Item onClick={goFirst}>First</Item>
+				<Item onClick={goPrevious}>Second</Item>
+			</Panel>
+		</Panels>
+	);
+};
+
+
+storiesOf('Panels', module)
+	.add(
+		'Panels',
+		() => (<BasicPanels />),
+		{
+			props: {
+				noScroller: true,
+				noPanels: true
+			},
+			text: 'The basic Panels'
+		}
+	);
+
+const MenuItemBase = kind({
+	name: 'MenuItem',
+
+	propTypes: {
+		css: PropTypes.object,
+		icon: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
+		path: PropTypes.string
+	},
+
+	defaultProps: {
+		disabled: false
+	},
+
+	styles: {
+		css: componentCss,
+		className: 'menuItem'
+	},
+
+	handlers: {
+		onClick: handle(
+			forward('onClick')
+		)
+	},
+
+	computed: {
+		iconBefore: ({icon}) => {
+			if (!icon) return;
+
+			return (<Button size="large" backgroundOpacity="lightOpaque" icon={icon} />);
+		}
+	},
+
+	render: ({css, iconBefore, ...rest}) => {
+		return (
+			<Item
+				css={css}
+				{...rest}
+				slotAfter={
+					<Icon size="small">arrowlargeright</Icon>
+				}
+				slotBefore={iconBefore}
+			/>
+		);
+	}
+});
+
+const MenuItem = Linkable(MenuItemBase);
+
+const MainPanel = kind({
+	name: 'MainPanel',
+
+	render: (props) => (
+		<Panel spotlightId="main-panel-container" {...props}>
+			<Scroller>
+				<MenuItem path="./page1">Pages</MenuItem>
+				<MenuItem path="./page2">General</MenuItem>
+			</Scroller>
+		</Panel>
+	)
+});
+
+const Page1 = kind({
+	name: 'Page',
+
+	render: (props) => (
+		<Panel spotlightId="page1-container" {...props}>
+			<Scroller>
+				<MenuItem path="./endPage">Page1</MenuItem>
+				<MenuItem path="./endPage">Page2</MenuItem>
+			</Scroller>
+		</Panel>
+	)
+});
+
+const Page2 = kind({
+	name: 'Page',
+
+	render: (props) => (
+		<Panel spotlightId="page2-container" {...props}>
+			<Scroller>
+				<MenuItem path="./endPage">Page1</MenuItem>
+				<MenuItem path="./endPage">Page2</MenuItem>
+			</Scroller>
+		</Panel>
+	)
+});
+
+const EndPage = kind({
+	name: 'Page',
+
+	render: () => (
+		<div>
+			<h1>EndPage</h1>
+		</div>
+	)
+});
```

Comment:
```suggestion
const EndPage = () => <h1>EndPage</h1>;
```

You can simply use functional component rather than using kind.
If it doesn't take advantage of kind's "computed" or etc, I prefer the simple one.
If you can apply other parts like this, please change that simple.

Reply:
I made the change, but for the other pages I left it with kind because it would break a bit the spotlight focus(when left is pressed it would select the close button instead of the left arrow)